### PR TITLE
PGPTool.PGPTool version 0.5.9.2

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107140)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package requires a `JRE`:
image

File: manifests/p/PGPTool/PGPTool/0.5.9.2/PGPTool.PGPTool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: PGPTool.PGPTool
+PackageVersion: 0.5.9.2
+PackageLocale: en-US
+Publisher: PGPTool
```
Comment (Trenly) [inline manifests/p/PGPTool/PGPTool/0.5.9.2/PGPTool.PGPTool.locale.en-US.yaml:7]:
```suggestion
Publisher: Sergey Karpushin
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
> This package requires a `JRE`: image

This is unchanged.

File: manifests/p/PGPTool/PGPTool/0.5.9.2/PGPTool.PGPTool.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: PGPTool.PGPTool
+PackageVersion: 0.5.9.2
```
Comment (stephengillie) [inline manifests/p/PGPTool/PGPTool/0.5.9.2/PGPTool.PGPTool.installer.yaml:5]:
```suggestion
PackageVersion: 0.5.9.2
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**